shared/profile/user-actions.js

```diff
@@ -194,17 +197,93 @@ const _DropdownButton = (props: DropdownProps & OverlayParentProps) => (
       position="bottom right"
       visible={props.showingMenu}
     />
-  </ClickableBox>
+  </Kb.ClickableBox>
 )
 
-const ellipsisIcon = platformStyles({
+export type StellarFederatedAddressProps = {|
+  currentlyFollowing?: boolean,
+  stellarAddress: string,
+  onSendOrRequest: (isRequest: boolean) => void,
+  onCopyAddress: () => void,
+|}
+
+export const MakeStellarAddressMenuItems = (props: StellarFederatedAddressProps) => [
```

Comment:
this isn't a component, so lets not uppercase it

Reply:
ok